### PR TITLE
Symbolic Navier-Stokes element test

### PR DESCRIPTION
**Pull request by rubenzorrilla**
Symbolic Navier-Stokes element LHS and RHS test (together with minor changes in the symbolic generation).

### CODE REVIEW

File: applications/FluidDynamicsApplication/CMakeLists.txt [hunk 1]
```diff
@@ -37,6 +36,8 @@ set( KRATOS_FLUID_DYNAMICS_APPLICATION_SOURCES
   ${CMAKE_CURRENT_SOURCE_DIR}/custom_constitutive/newtonian_2d_law.cpp
   ${CMAKE_CURRENT_SOURCE_DIR}/custom_constitutive/newtonian_3d_law.cpp
   ${CMAKE_CURRENT_SOURCE_DIR}/custom_constitutive/herschel_bulkey_3d_law.cpp
+
```
Comment (RiccardoRossi) [inline applications/FluidDynamicsApplication/CMakeLists.txt:39]: i think this should be in only if tests are compiled

**Review by RiccardoRossi — APPROVED** [review]
just ensure that compilation is conditional (if tests are applied). other than that go ahead

**Comment by loumalouomega** [comment]
NICE WORK

File: applications/FluidDynamicsApplication/CMakeLists.txt (hunk 1 above)
Comment (rubenzorrilla) [reply applications/FluidDynamicsApplication/CMakeLists.txt:39]: Totally agree. I will do it asap. Thanks @RiccardoRossi .

**Review by rubenzorrilla — COMMENTED** [review]
@roigcarlo I think that this is ok, but would you mind to have a look? Thanks!

**Comment by roigcarlo** [comment]
Yup that's it :+1:

**Pull request merged**